api/v1alpha1/resourcegraphdefinition_types.go

```diff
@@ -86,11 +90,24 @@ type Schema struct {
 	// Validation is a list of validation rules that are applied to the
 	// resourcegraphdefinition.
 	Validation []Validation `json:"validation,omitempty"`
-	// AdditionalPrinterColumns defines additional printer columns
-	// that will be passed down to the created CRD. If set, no
-	// default printer columns will be added to the created CRD,
-	// and if default printer columns need to be retained, they
-	// need to be added explicitly.
+	// AdditionalPrinterColumnPolicy defines additional printer columns
+	// that will be passed down to the created CRD. The Default tag is "Replace", that only uses the provided columns, "Add:" tag Merge provided columns with defaults.
+	//
+	// AdditionalPrinterColumnPolicy controls how provided additional printer
+	// columns are applied to the generated CRD.
+	//
+	// - "Replace": only use the provided columns; if none are provided, fall
+	//   back to the operator defaults (backwards compatible default).
```

Comment:
```suggestion
	//   back to the default columns.
```

Also: Please insert the default columns here so they are documented